test/verify/check-superuser

```diff
@@ -69,6 +69,12 @@ class TestSuperuser(testlib.MachineCase):
         self.restore_file("/var/lib/sudo/lectured/admin")
         m.execute("rm -rf /var/{db,lib}/sudo/lectured/admin")
 
+        # Sudo since 1.9.15 uses a UID not a username https://www.sudo.ws/releases/stable/#1.9.15
+        uid = m.execute("id -u admin").strip()
```

Comment:
I wonder why we don't simply restore_dir("/var/db/sudo/lectured") and remove all files in it.

Reply:
Ah, good question, happy to do that. Seems the test flakes a bit so I will retry it and then do a follow up once the image is merged